### DISCUSSION TITLE
Why does one block change update every other block's changedfieldindicator?

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#12597] [General]
I've just added some field instructions to one of the fields inside our site's main Matrix field. In Project Config, I would expect this to update the project.yaml file with a dateModified, and add the new field instruction to that block's yaml file. Instead, every single block in the matrix field updates it's changedFieldIndicator, which both makes migrations and deployment take longer, and makes it difficult to find what changed in a commit.

It didn't always do this - is this behaviour new?

![image](https://user-images.githubusercontent.com/1250017/215500260-cb0f5214-e50e-4cbf-b7e6-d545dbe8d164.png)

**Comment by brandonkelly** [comment]
Craft doesn’t have any `changedFieldIndicator` properties anywhere, so I’m not sure where that’s coming from or why it would be getting updated.

**Comment by darylknight** [comment]
Ah, thank you - sorry about that. Yeah, it looks like it's caused by Super Table.
